Project.toml

```
@@ -1,6 +1,6 @@
 name = "Compat"
 uuid = "34da2185-b29b-5c13-b0c7-acf172513d20"
-version = "3.8.0"
+version = "3.9.0"
```

```suggestion
version = "3.9.0-DEV"
```
I'll try to get #693, #695 and this one in soonish, and then do a new version, instead of bumping the version three times in a row, especially as that would mean having to update the not-yet-merged PRs after every version bump.